content/zh/docs/concepts/cluster-administration/logging.md

```diff
@@ -0,0 +1,370 @@
+* 使用在每个节点上运行的节点级日志记录代理。
+<!-- * Include a dedicated sidecar container for logging in an application pod. -->
+* 在Pod中使用专用于记录日志的边车容器。
+<!-- * Push logs directly to a backend from within an application. -->
+* 将日志直接从应用程序中推送到日志记录后端。
+
+<!-- ### Using a node logging agent -->
+### 使用节点日志记录代理
+
+![Using a node level logging agent](/images/docs/user-guide/logging/logging-with-node-agent.png)
+
+<!-- You can implement cluster-level logging by including a _node-level logging agent_ on each node. The logging agent is a dedicated tool that exposes logs or pushes logs to a backend. Commonly, the logging agent is a container that has access to a directory with log files from all of the application containers on that node. -->
+您可以通过在每个节点上使用 _节点级的日志记录代理_ 来实现群集级日志记录。日志记录代理是一种用于暴露日志或将日志推送到后端的专用工具。通常，日志记录代理程序是一个容器，它可以访问包含该节点上所有应用程序容器的日志文件的目录。
+
+<!-- Because the logging agent must run on every node, it's common to implement it as either a DaemonSet replica, a manifest pod, or a dedicated native process on the node. However the latter two approaches are deprecated and highly discouraged. -->
+由于日志记录代理必须在每个节点上运行，它可以用DaemonSet副本，Pod 或 本机进程来实现它的。然而，后两种方法被弃用并且非常不别推荐。
+
+<!-- Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, because it creates only one agent per node, and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_. -->
+对于Kubernetes集群，使用节点级日志记录代理是最常见和鼓励的方法，因为它每个节点只创建一个代理，并且不需要对节点上运行的应用程序进行任何更改。然而，节点级日志记录 _仅适用于应用程序的标准输出和标准错误_。
+
+<!-- Kubernetes doesn't specify a logging agent, but two optional logging agents are packaged with the Kubernetes release: [Stackdriver Logging](/docs/user-guide/logging/stackdriver) for use with Google Cloud Platform, and [Elasticsearch](/docs/user-guide/logging/elasticsearch). You can find more information and instructions in the dedicated documents. Both use [fluentd](http://www.fluentd.org/) with custom configuration as an agent on the node. -->
+虽然Kubernetes中没有指定日志代理，但是两个可选的日志代理与Kubernetes版本一起打包：用于Google Cloud Platform的[Stackdriver Logging](/docs/user-guide/logging/stackdriver)和[Elasticsearch](/docs/user-guide/logging/elasticsearch)。您可以在专用文档中找到更多信息和说明。两者都使用了自定义的配置的[fluentd](http://www.fluentd.org/)作为节点上的代理。
+
+<!-- ### Using a sidecar container with the logging agent -->
+### 使用带有日志记录代理的边车容器
+
+<!-- You can use a sidecar container in one of the following ways: -->
+您可以通过以下方式使用边车容器：
+
+<!-- * The sidecar container streams application logs to its own `stdout`. -->
+* sidecar容器将应用程序日志流式传输到自己的`stdout`。
+<!-- * The sidecar container runs a logging agent, which is configured to pick up logs from an application container. -->
+* sidecar容器运行日志记录代理程序，该代理程序配置为从应用程序容器中获取日志。
+
+<!-- #### Streaming sidecar container -->
+#### Streaming到sidecar容器
+
+![Sidecar container with a streaming container](/images/docs/user-guide/logging/logging-with-streaming-sidecar.png)
```

Comment:
This alt text is untranslated. I recommend translating it.

Reply:
it's always hard to translate "streaming ..."

Reply:
just leave it as it is, no need to translate it